Gemfile

```diff
@@ -43,9 +43,11 @@ gem 'sass-rails', '~> 6.0'
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
+  gem 'apitome', '~> 0.3.1'
```

Comment:
It's ok to load this gem in development environments only, you just have to adjust the code [like I mentioned before in this comment](https://github.com/LeandroSoria2/TargetApi/pull/19#discussion_r920245084)